app/services/email_reply_service.rb

```
@@ -0,0 +1,324 @@
+        errors: answer.errors.full_messages
+      })
+      return { success: false, error: error_msg }
+    end
+  rescue => e
+    # Catch unexpected errors during processing
+    error_msg = "Unexpected error: #{e.message} - Backtrace: #{e.backtrace.first(5).join("\n")}"
+    update_inbound_email(status: 'failed', notes: error_msg)
+    Rails.logger.error error_msg
+    # Optionally, notify admins (e.g., via Sentry, Airbrake, or email)
+    return { success: false, error: error_msg }
+  end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for process is too high. [<12, 64, 10> 65.88/23]

(Metrics/AbcSize)